frame/staking-rewards/src/lib.rs

```diff
@@ -65,10 +100,22 @@ pub mod pallet {
 		type Balance: Parameter + Member + BalanceLike + FixedPointOperand;
 
 		/// The reward pool ID type.
-		type PoolId: Parameter + Member + Clone + FullCodec;
+		/// Type representing the unique ID of a pool.
+		type PoolId: FullCodec
```

Comment:
Here is PoolId and it is clear it staking pool fron mod name, same should be in traits